.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true
+contact_links:
+- name: 🙋 Use Questions
```

Comment:
Maybe "Usage" instead of "Use"? It took me a while to understand that "use" was not a verb here.